### PR TITLE
Fix button focus boder cut off

### PR DESCRIPTION
**Pull request by coolcat-creations**
Pull Request for Issue #37706.

### Summary of Changes
I added a 0.25 rem distance to the left of the toolbar so that the focus border is not cut off.

### Testing Instructions
Apply patch and in Cassiopeia Frontend in the Media Modal window the Upload Button should have a full focus border

### Actual result BEFORE applying this Pull Request
grafik

### Expected result AFTER applying this Pull Request
grafik

### Documentation Changes Required
No

### CODE REVIEW

**Comment by brianteeman** [comment]
Sorry but this is not correct as it wont work on an rtl site and it's so generic that it will have an impact everywhere.

Pretty sure that the correct fix is to change the padding in the subhead class below

https://github.com/joomla/joomla-cms/blob/d43e5a576cafd61cd4d32f5b2e2beae39637d7de/build/media_source/templates/site/cassiopeia/scss/blocks/_toolbar.scss#L1-L12

changing from `padding: 10px 0` to simply `padding: 10px` should do it

**Comment by coolcat-creations** [comment]
Ok, thanks for the suggestion I changed it

File: build/media_source/templates/site/cassiopeia/scss/blocks/_toolbar.scss [hunk 1]
```diff
@@ -6,11 +6,12 @@
   z-index: $zindex-toolbar;
   width: auto;
   min-height: 43px;
-  padding: 10px 0;
+  padding: .25rem;
   color: #495057; //#0c192e;
   background: $white;
   box-shadow: -3px -2px 22px #ddd;
 
+
```
Comment (sandewt) [inline build/media_source/templates/site/cassiopeia/scss/blocks/_toolbar.scss]: Why added an empty line?

Comment (coolcat-creations) [reply build/media_source/templates/site/cassiopeia/scss/blocks/_toolbar.scss]: Thanks, fixed. lint did not see my mistake and neither I :-)

**Comment by crystalenka** [comment]
I have tested this item :white_check_mark: successfully on edbe8d2604187ad81715cfef9a80e6e1875e0f3f<br /><br />Fixes the focus border for me. :) thanks!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37707">issues.joomla.org/tracker/joomla-cms/37707</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on edbe8d2604187ad81715cfef9a80e6e1875e0f3f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37707">issues.joomla.org/tracker/joomla-cms/37707</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37707">issues.joomla.org/tracker/joomla-cms/37707</a>.</sub>

**Comment by pritam825** [comment]
I have tested this item :white_check_mark: successfully on edbe8d2604187ad81715cfef9a80e6e1875e0f3f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37707">issues.joomla.org/tracker/joomla-cms/37707</a>.</sub>

**Comment by laoneo** [comment]
Thanks!

**Pull request merged**